api/e2e/test/01_create_router_test.go

```diff
@@ -4,12 +4,12 @@ package e2e
 
 import (
 	"fmt"
+	"github.com/gojek/turing/api/turing/service"
```

Comment:
Normally, the `goimports` linter would complain about this ordering. I just realised that `make lint` doesn't include the e2e directory - `golangci-lint` only works when there is a .mod file in the root, for the files in that module and the code is compilable. Not sure that there is a way to get this working for the e2e folder at the moment.

I usually have goreturns or goimports enabled in the IDE for auto-formatting on save file.

Reply:
I've updated the golangci.yml config file to include Go source with e2e build tags, so the linting will run on e2e package as well.

Reply:
I didn't realise this was possible. Thanks!